### PR TITLE
✨ app: add withdrawal processing view

### LINKED ISSUE #165: ui: modification on withdraw proposals management

### PR DESCRIPTION
**Pull request by franm91**
closes #165

### CODE REVIEW

**Review by dieguezguille — APPROVED** [review]

**Review by cruzdanilo — CHANGES_REQUESTED** [review]

File: src/components/home/Processing.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+import assetLogos from "../../utils/assetLogos";
+import useMarketAccount from "../../utils/useMarketAccount";
+import ActionButton from "../shared/ActionButton";
+import AssetLogo from "../shared/AssetLogo";
+import SafeView from "../shared/SafeView";
+import Spinner from "../shared/Spinner";
+import Text from "../shared/Text";
+import View from "../shared/View";
+
+type Proposal = [amount: bigint, market: `0x${string}`, recipient: `0x${string}`, timestamp: bigint];
```
[the remark this answers is not in the document]
Comment (franm91) [reply src/components/home/Processing.tsx]: fixed

File: src/components/home/Processing.tsx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        if (proposal?.[0] === 0n) return 0;
+        if (proposal?.[3]) return 60_000;
+        const proposalTimestamp = Number(proposal?.[3]) * 1000;
+        const proposalEndTime = proposalTimestamp + 5 * 60 * 1000;
+        const timeLeft = proposalEndTime - Date.now() + 1000;
+        return Date.now() > proposalEndTime ? 2000 : timeLeft;
+      },
+      gcTime: 0,
+    },
+  }) as { data: Proposal | undefined };
```
[the remark this answers is not in the document]
Comment (franm91) [reply src/components/home/Processing.tsx]: fixed

File: src/components/home/Processing.tsx [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    args: [address ?? zeroAddress],
+    query: {
+      refetchInterval: () => {
+        if (proposal?.[0] === 0n) return 0;
+        if (proposal?.[3]) return 60_000;
+        const proposalTimestamp = Number(proposal?.[3]) * 1000;
+        const proposalEndTime = proposalTimestamp + 5 * 60 * 1000;
+        const timeLeft = proposalEndTime - Date.now() + 1000;
+        return Date.now() > proposalEndTime ? 2000 : timeLeft;
+      },
```
[the remark this answers is not in the document]
Comment (franm91) [reply src/components/home/Processing.tsx]: fixed

**Review by dieguezguille — APPROVED** [review]
LGTM now 🆗

**Review by dieguezguille — DISMISSED** [review]

**Review by cruzdanilo — APPROVED** [review]

**Pull request merged**